File: TimBola.php
<?php

class TimBola{
    private $namaTim;
    private $negara;
    private $tahunBerdiri;
    private $pemain = array();
   

    public function __construct($namaTim = 0, $negara = 0, $tahunBerdiri = 0, $pemain = 0) {
        $this->namaTim = $namaTim;
        $this->negara = $negara;
        $this->tahunBerdiri = $tahunBerdiri;
        $this->pemain = $pemain;
        

    }

    public function setNamaTim($namaTim) { //
        $this->namaTim = $namaTim;
    }
    public function getNamaTim(){ 
        return $this->namaTim;
    }

    public function setNegara($negara) { //
        $this->negara = $negara;
    }

    public function getNegara(){ 
        return $this->negara;
    }

    public function setTahunBerdiri($tahunBerdiri){
        $this->tahunBerdiri = $tahunBerdiri;
    }
    public function getTahunBerdiri(){
        return $this->tahunBerdiri;
    }

    public function setPemain($pemain){
        $this->pemain = $pemain;
    }

    public function getPemain(){
        return $this->pemain;
    }

    
} 




?>

File: TimBola.cpp
#include <iostream>
#include <string> 


using namespace std;

class TimBola{
    private:
    string namaTim;
    string negara;
    string tahunBerdiri;
    

    public:
    TimBola(){
    }
    
    class DaftarPemain{
    	private:
	    string pemain;

    	public:
    	DaftarPemain(){
    	}
	
	DaftarPemain(string p){
		pemain = p;
	}
	
	 void setPemain(string p){
        pemain = p;
    }
    
    string getPemain(){
        return pemain;
    }
	
		~DaftarPemain(){
		}
	};

    TimBola(string nT, string n, string tB){
        namaTim = nT;
        negara = n;
        tahunBerdiri = tB;
    
    }

    void setNamaTim(string nT){
        namaTim = nT;
    }

    void setTahunBerdiri(string tB){
        tahunBerdiri = tB;
    }

    void setNegaraTim(string n){
        negara = n;
    }

   

    string getNamaTim(){
        return namaTim;
    }

    string getTahunBerdiri(){
        return tahunBerdiri;
    }

    string getNegaraTim(){
        return negara;
    }

   
    ~TimBola(){
    }

};
